CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+First of all, read how to deploy a local copy of Pan Docs in the [README](https://github.com/gbdev/pandocs#contributing).
```

Comment:
Given that this document mentions the project board as well, maybe we should mention that we accept contributions that aren't just code? Input, feedback, and [sometimes graphical design](https://github.com/gbdev/pandocs/issues/371) are useful as well.